builtin/staker/validation/service.go

```diff
@@ -203,8 +208,12 @@ func (s *Service) SetBeneficiary(validator thor.Address, validation *Validation,
 }
 
 func (s *Service) DecreaseStake(validator thor.Address, validation *Validation, amount uint64) error {
+	var overflow bool
 	if validation.Status == StatusActive {
-		validation.PendingUnlockVET += amount
+		validation.PendingUnlockVET, overflow = math.SafeAdd(validation.PendingUnlockVET, amount)
```

Comment:
This is not required, as upstream already checked `amount <= LockedVET - PendingUnlockVET`, max value is limited to `LockedVET`.